src/www/ui/api/documentation/openapi.yaml

```diff
@@ -1763,40 +1771,72 @@ paths:
                   $ref: '#/components/schemas/Group'
         default:
           $ref: '#/components/responses/defaultResponse'
-
-  /groups/{id}/members:
+  /groups/{groupId}/user/{userId}:
     parameters:
-      - name: id
+      - name: groupId
         in: path
         required: true
-        description: ID of the group
+        description: Group id
         schema:
-          type: integer
-    get:
-      operationId: getGroupUsersWithRoles
+          type: string
+      - name: userId
+        in: path
+        required: true
+        description: User Id
+        schema:
+          type: string
+    put:
+      operationId: updatePermissionByGroupIdAndUserId
       tags:
+        - Users
         - Groups
-      summary: Get the users with their roles from a group
+      summary: Update Permission
       description: >
-        Returns accessible objects of users with roles from the group
+        Update User's permission in group.
+      requestBody:
+        description: Value of the new permission
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/NewGroupPermission'
```

Comment:
Looks like definition for `components/schemas/NewGroupPermission` is missing in the yaml file.

Reply:
Something like following will be required:
```diff
diff --git a/src/www/ui/api/documentation/openapi.yaml b/src/www/ui/api/documentation/openapi.yaml
index 254f16ebf..8c0612c34 100644
--- a/src/www/ui/api/documentation/openapi.yaml
+++ b/src/www/ui/api/documentation/openapi.yaml
@@ -3007,6 +3007,19 @@ components:
               enum:
                 - OK
                 - ERROR
+    NewGroupPermission:
+      description: User Group permission update
+      type: object
+      properties:
+        perm:
+          type: integer
+          description: |
+            New user permission in the group.
+            0: User
+            1: Admin
+            2: Advisor
+          minimum: 0
+          maximum: 2
   responses:
     defaultResponse:
       description: Some error occured. Check the "message"
```

Reply:
Done 👍